server/resource/columns.go

```diff
@@ -83,6 +83,9 @@ var StandardRelations = []api2go.TableRelation{
 	api2go.NewTableRelation("mail_box", "belongs_to", "mail_account"),
 	api2go.NewTableRelation("mail", "belongs_to", "mail_box"),
 	api2go.NewTableRelationWithNames("task", "task_executed", "has_one", USER_ACCOUNT_TABLE_NAME, "as_user_id"),
+	api2go.NewTableRelationWithNames("calendar", "calendar", "has_one", USER_ACCOUNT_TABLE_NAME, "as_owner_id"),
+	api2go.NewTableRelationWithNames("collection", "collection", "has_one", USER_ACCOUNT_TABLE_NAME, "as_owner_id"),
```

Comment:
all entities (calendar/collection) already have the relation "belongs_to" user_account (with user_account_id as fk column), so this as_owner_id is probably not needed. use that for ownership.